docs/deployment_groups.md

```diff
@@ -11,15 +11,23 @@ motivation.
 
 ### Create
 
-Create a deployment group by specifying the [host labels][1] to filter on. The command below will
-create a deployment group whose hosts match all the specified labels.
+A host selector is an expression of the form <label-pattern> <operand> <operand pattern>.
```

Comment:
What's up with all the "patterns"?

The left hand side is always a label, no "patterns" allowed. The right hand side is always an operand.